### ISSUE TITLE
Error unzipping simplelocalize.zip

### ISSUE THREAD

**Issue by marques-p44** [simplelocalize/simplelocalize-cli#20]
I'm receiving this error when I run the curl command below. It looks like the hosted zip file is corrupt? Is anyone else seeing this problem?

```
➜ curl -s https://get.simplelocalize.io | bash
--> Fetching CLI from https://github.com/simplelocalize/simplelocalize-cli/releases/download/0.5.1/simplelocalize-cli-macos.zip
--> Extracting archive
[/Users/me/.simplelocalize/simplelocalize.zip]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of /Users/me/.simplelocalize/simplelocalize.zip or
        /Users/me/.simplelocalize/simplelocalize.zip.zip, and cannot find /Users/me/.simplelocalize/simplelocalize.zip.ZIP, period.
```

**Comment by mykong89** [comment]
I also got this error but it works anyway. 

I have `unzip` installed
```
mike@macbook:~ $ unzip --help
UnZip 6.00 of 20 April 2009, by Info-ZIP.  Maintained by C. Spieler.  Send
bug reports using http://www.info-zip.org/zip-bug.html; see README for details.

Usage: unzip [-Z] [-opts[modifiers]] file[.zip] [list] [-x xlist] [-d exdir]
  Default action is to extract files in list, except those in xlist, to exdir;
  file[.zip] may be a wildcard.  -Z => ZipInfo mode ("unzip -Z" for usage).

  -p  extract files to pipe, no messages     -l  list files (short format)
  -f  freshen existing files, create none    -t  test compressed archive data
  -u  update files, create if necessary      -z  display archive comment only
  -v  list verbosely/show version info       -T  timestamp archive to latest
  -x  exclude files that follow (in xlist)   -d  extract files into exdir
modifiers:
  -n  never overwrite existing files         -q  quiet mode (-qq => quieter)
  -o  overwrite files WITHOUT prompting      -a  auto-convert any text files
  -j  junk paths (do not make directories)   -aa treat ALL files as text
  -C  match filenames case-insensitively     -L  make (some) names lowercase
  -X  restore UID/GID info                   -V  retain VMS version numbers
  -K  keep setuid/setgid/tacky permissions   -M  pipe through "more" pager
See "unzip -hh" or unzip.txt for more help.  Examples:
  unzip data1 -x joe   => extract all files except joe from zipfile data1.zip
  unzip -p foo | more  => send contents of foo.zip via pipe into program more
  unzip -fo foo ReadMe => quietly replace existing ReadMe if archive file newer
```

**Comment by jpomykala** [comment]
@mykong89 Thank guys, I'm on it!

**Comment by jpomykala** [comment]
Fix is on production. No changes are needed on your side! 💯 
I also released version 0.5.3 which includes latest changes.

https://github.com/simplelocalize/simplelocalize-cli/releases/tag/0.5.3
